BREAKING_CHANGES.md

```diff
@@ -4,6 +4,19 @@ ChameleonForms Breaking Changes
 Version 2.0.0
 =============
 
+Deprecated `WithoutLabel` method on `IFieldConfiguration`. It still works (for now), but the method has been marked with the `[Obsolete]` attribute.

```

Comment:
This is from a previous PR, but I noticed that it was missing so added in.

I suspect we can release v2 after this PR is merged in. Thoughts @MattDavies?